### DISCUSSION TITLE
RFC: Proposal for Establishing KtonDAO Using KTON Tokens

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1512] [General]
**Background and Motivation**

The idea of creating KtonDAO, a DAO governed by KTON tokens, originated from a discussion initiated by DCDAO member Saso ([discussion link](https://github.com/orgs/dcdao/discussions/32)). Recent technological upgrades, including the separation of KTON Staking, updates to the design of RingDAO, and discussions on the token economy of RING and KTON, have made this an opportune time to propose the establishment of KtonDAO.

**Core Proposal**

KtonDAO will utilize KTON tokens as governance tokens. Deposit module and Kton Staking module will be included and owned by KtonDAO, they will maintain its current logic and operations, but the ownership over the Deposit and Kton Staking, as well as the minting and burning of KTON, will be transferred to KtonDAO.

**Formalization of Contracts and Proposals between KtonDAO and RingDAO**

Historically, KTON and RING deposits have a long-standing presence in the Darwinia community. Previously, KTON was seen as a derivative token of RING without a thorough analysis of the underlying token economic relationship. The establishment of KtonDAO will formalize and clarify this relationship through following proposal or contract:

- KtonDAO creates Deposit contract where locking RING for different fixed terms will result in the immediate issuing of a certain amount of KTON as prepaid interest to incentivize long-term RING commitment.
- As a reward for this proposal and contribution, RingDAO will allocate 20% of its issuance to KtonDAO, which is currently distributed to KTON Staking participants via Kton Staking.

**Technical and Permission Changes**

1. The Deposit contract and Kton Staking will be under KtonDAO.
2. The final authority for minting and burning KTON will transfer to KtonDAO, previously limited to the Deposit contract.
3. The 20% issuance reward from RingDAO will be allocated to the KtonDAO Treasury before distribution to KtonStaking participants. Though KtonDAO's treasury balance is zero at the beginning, but KtonDAO may govern to make changes in Treasury strategy, including reducing staking incentives, accumulating RING, or implementing a buyback strategy similar to MakerDAO's buyback, using excess RING to buy back KTON and create liquidity LP on exchanges like Snowswap. Reference: [MakerDAO buyback](https://makerburn.com/#/buyback).
4. KtonDAO will become independent and may upgrade and modify the above contracts and proposals. Governance considerations originate from their own token economy naturely.

**Conclusion**

Establishing KtonDAO will transfer governance authority to the community, fostering decentralization and bringing vitality and potential to the community.

Previous DCDAO practices based on Tally have provided valuable experience, and the necessary infrastructure and technological feasibility are well-established. Looking forward to experienced community members like @hujw77  helping to drive the development and implementation of this proposal. cc @AurevoirXavier @wuminzhe @boundless-forest

**Comment by SasoLithops** [comment]
Hi!

Glad to see this moving forward. I fully support the idea behind proposal and look forward to new opportunities for KTON.

I however one question.

  _"the ownership over the Deposit and Kton Staking, as well as the minting and burning of KTON, will be transferred to KtonDAO."_

I always fear when minting of new tokens is transfered to DAO. Sadly many people don't follow governance and don't vote and in case some people try to take over DAO, could we have some kind of security set so that does not happen. Like when it comes to changing contract quorum should be higher?

**Reply by Laki389** [reply]
>     1. _"As a reward for this proposal and contribution, RingDAO will allocate 20% of its issuance to KtonDAO, which is currently distributed to KTON Staking participants vis Kton Staking."_
>        _"The 20% issuance reward from RingDAO will be allocated to the KtonDAO Treasury before distribution to KtonStaking _participants"__
> 
> 
>     *** If I understand correctly KTON DAO treasury would fund from Kton Staking Pool (which gets 20% of all new minted Ring so KTON DAO treasury would get 4% of total newly minted Ring and stakers 16%)?**

Hi @SasoLithops 
I understand this like: currently, 60% goes to Darwinia Treasury, 20% to Ring staking and 20% to Kton staking.

The proposal in the discussion envisages that this 20% that currently goes to kton staking, will go to the KtonDAO Treasury and further as the members of the DAO decide.

**Reply by SasoLithops** [reply]
Thank you Laki. You are right. I misunderstood. This makes more sense. Will edit it out of my reply.

**Comment by danairey** [comment]
Like this idea, with the low market cap of kton it has unlimited potential. We need to find other use cases for it. Obviously its  great as a staking token to earn ring but something different to set it on its way.

**Comment by johneggleston** [comment]
I think one of the key things to focus on is dialling back the issuance of KTON. At the moment both RING and KTON are very inflationary so if one of the tokens in this dual-token system could have a deflation rate that exceeds the issuance rate, that would be a much more attractive system for investors. 

An alternate approach which would probably be easier in the long run is to keep KTON inflationary, but make RING the deflationary asset. I think this would be a much better approach for 2 reasons. 

1. RING has a fixed maximum supply which makes burning/deflation far more manageable and impactful. 

2. Keeping the supply of a governance limited would increase monopolisation over the voting power, but adding inflationary pressure would mean that people who want to keep their votes heavy would have to make sure that their KTON bags are growing at the same rate as the supply is inflating. 

So if KTON is used as the governance token and remains inflationary asset with no fixed supply, I’d recommend a burn-to-vote system to combat monopolisation. 

Hope you like this input.

**Comment by SasoLithops** [comment]
Nice to see engagement going. Any ideas more then welcome.

@johneggleston I don't see KTON as very inflationary as only way to mint it is locking your RING. I think it is just that it has no other value next to staking once minted and that gives it small value. 
At this moment my opinion is I would not be in favour of burning nor minting new KTON. I think we should put the focus into how to build a sustainable treasury first so we can move forward with additional usecases. Agree with @danairey statement.
I am playing with one idea (still in research phase) that could help in KTON scarcity and bring different tokens into treasury. But this idea would need some input at start as well. Will let you all know more on this once I get more answers if this is possible.

**Comment by Laki389** [comment]
>  At this moment my opinion is I would not be in favour of burning nor minting new KTON.

**This part of @SasoLithops  comment inspired me to imagine a scenario where no new KTON is minted.
What if there is no minting of new KTON in the future?** 

Darwinia has a dual system where locking a Ring for a certain number of months earns/mints KTON which is immediately available.
This is KTON inflation which is not fixed, cannot be predicted and depends entirely on the amount of newly locked Ring, which cannot be predicted.

What if there is no new KTON?

**Negative aspects:** :weary:

- If there is no new KTON mint, the dual system collapses, and it is logical that part of the inflation of Ring, or 20% which currently goes to Kton staking, will stop being used for this in the future.
What would be done with that 20% of the newly created Ring would depend entirely on Ring governance.
There are more scenarios for that, but here the topic is about Kton so this is not a topic of discussion.

**Positive aspects:** :smiley: 

- Kton, according to Coingecko has only **128 830** Total Supply. This amount will be forever, unless some burning mechanism is created, then it would even be deflationary.

- Kton, according to Coingecko, currently has only **$473,663** mcap.
Its growth and decline will not be dependent on and linked to the Ring.
Under the current system, for Kton to have 47M mcap, or 100x, Ring should also have, say, $650M mcap.
As envisioned, in theory, Kton could have an even higher mcap than Ring, which is currently not possible. 
Currently, because locking in 10 000 Ring for 36 months immediately gives 3.6446 Kton, it is not possible for 1 Kton to be worth more than 2 744 Ring.
In the past, this fact stifled any attempt to pump Kton.
Whenever there has been a Kton pump in the past, there are people who see a benefit in buying a Ring, locking it for 36 months, minting Kton, selling it and getting $ similar to the initial amount. By the way, they have Ring locked and staked, which has 13-14% APY. After 36 months their Ring is free.
Summary, every Kton pump attempt fails, makes a small Ring pump.


Further development and evolution would depend entirely on Kton governance.
It could become ktonwifhat 😝 :sweat_smile: , the first Darwinia meme (even though it's Apo :grin:).
In theory, it could even move to Solana :sweat_smile: or other more hot/hyped chains than Polkadot is.

**Reply by SasoLithops** [reply]
Just quick reply glad you were inspired, but what I meant to say was I am not for additional minting,  only what already exists by locking Ring😉 My YugoEnglish strikes again🫣

**Reply by SasoLithops** [reply]
And another thought what if we introduce locking Kton. Or burning some to receive something else( Laki already said I am thinking in Evolution Land)  
Like it was meant at first, that developers on Darwinia chain would need KTON and burn it. 
Will think more....thank you for brainstorming @Laki389 @johneggleston 💪

**Comment by hackfisher** [comment]
Engineering progress update: https://github.com/darwinia-network/kton-staker/issues/10

**Comment by hackfisher** [comment]
For experiment on Crab: https://www.tally.xyz/gov/cktondao

**Comment by hackfisher** [comment]
https://www.tally.xyz/gov/ktondao
